roles/cfg_openwrt/tasks/render_etc.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Create /etc directories part one
+  tags: always
+  file:
+    path: "{{ config_path }}/etc/{{ item.path }}"
+    state: directory
+  with_filetree: "{{ role_path }}/templates/{{ role }}"
+  when: item.state == 'directory'
+
+- name: Create /etc directories part two
+  tags: always
+  file:
+    path: "{{ config_path }}/etc/{{ item.path }}"
+    state: directory
+  with_filetree: "{{ role_path }}/files/{{ role }}"
+  when: item.state == 'directory'
+
+- name: Render /etc templates
+  tags: always
+  template: src={{ item.root }}/{{ item.path }} dest={{ config_path }}/etc/{{ item.path | regex_replace('\.j2$','') }}
```

Comment:
Using `{{item.root}}/{{item.path}}` instead of just `{{item.src}}` here because if item is a symlink, `item.src` contains the path to the symlink's target, which for some reason is not what we need here